Voycar.Api.Web.Tests.Integration/Users/Post_Login/Tests.cs

```diff
@@ -0,0 +1,206 @@
+namespace Voycar.Api.Web.Tests.Integration.Users.Post_Login;
+
+using Context;
+using Setup;
+using R = Features.Users.Endpoints.Post.Login;
+
+
+public sealed class State : StateFixture
+{
+    public Guid PlanId { get; set; }
+    public const string PlanName = "basic";
+}
+
+
+public class Tests : TestBase<App, State>
+{
+    private readonly App _app;
+    private readonly State _state;
+    private readonly VoycarDbContext Context;
+
+
+    // Setup request client
+    public Tests(App app, State state)
+    {
+        this._app = app;
+        this._state = state;
+        this.Context = this._app.Context;
+        this._state.PlanId = this.Context.Plans.First(p => p.Name == State.PlanName).Id;
+    }
+
+    public Features.Users.Endpoints.Post.Register.Request CreateValidRequest()
+    {
+        return new Features.Users.Endpoints.Post.Register.Request
+        {
+            Email = "unverified@test.de",
+            Password = "notsafe987",
+            FirstName = "null",
+            LastName = "null",
+            Street = "null",
+            HouseNumber = "null",
+            PostalCode = "null",
+            City = "null",
+            Country = "null",
+            BirthDate = new DateOnly(2002, 12, 12),
+            BirthPlace = "null",
+            PhoneNumber = "null",
+            PlanId = this._state.PlanId
+        };
+    }
+
+    [Fact]
+    public async Task Post_Request_ReturnsBadRequest_DueToUnverifiedUser()
+    {
+        // Arrange
+        var request = new R.Request { Email = "unverified@test.de", Password = "notsafe987" };
+
+        await this._app.Client
+            .POSTAsync<Features.Users.Endpoints.Post.Register.Endpoint, Features.Users.Endpoints.Post.Register.Request>(
+                this.CreateValidRequest()); // Register new user without verifying
+
+        // Act
+        var httpResponse = await this._app.Client.POSTAsync<R.Endpoint, R.Request>(request);
+
+        // Assert
+        httpResponse.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+
+
+    [Fact]
+    public async Task Post_Request_ReturnsBadRequest_DueToIncorrectPassword()
+    {
+        // Arrange
+        var memberClient =
+            await ClientFactory.CreateMemberClient(this._app, this.Context, "member@test.de", "password");
+        var request = new R.Request { Email = "member@test.de", Password = "diffPassword" };
+
+        // Act
+        var firstHttpResponse = await memberClient.PostAsync("/auth/logout", default); // Logout
+        var secondHttpResponse = await this._app.Client.POSTAsync<R.Endpoint, R.Request>(request); // Login
+
+        // Assert
+        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
```

Comment:
Da die Überprüfung von diesem Statuscode nicht direkt aus dem Namen der Testmethode hervorgeht, würde ich noch ein because angeben.
```suggestion
        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK,
            "factory method produced logged in user who should be able to log out");
```

Reply:
ah gut zu wissen, wusste nicht, dass das geht